cmd/dbfactory/cluster.go

```diff
@@ -79,3 +84,67 @@ func printJSON(data interface{}) error {
 	encoder.SetIndent("", "    ")
 	return encoder.Encode(data)
 }
+
+// searchOpts the options to be used for the search command
+type searchOpts struct {
+	tags map[string]string
+}
+
+// newSearchCommand adds a subcommand in cluster in order to be
+// able to search by given tags.
+// - Example -
+// dbfactory cluster -t 'DatabaseType=multitenant-rds'
+func newSearchCommand() *cobra.Command {
```

Comment:
vvv MInor: usually namings like `new` potentially leads to `newNew` and other ugly things :)

Reply:
It's based on this https://golang.org/doc/effective_go#composite_literals but it's up to u to avoid these during review time.